devops/ansible/roles/php70/vars/main.yml

```diff
@@ -9,7 +9,7 @@ debug_mode: false
 # allow_url_fopen
 #   Default Value: On
 php_allow_url_fopen: "Off"
-php_disable_functions: "exec,passthru,shell_exec,system,proc_open,popen"
+php_disable_functions: "exec,passthru,shell_exec,system,popen"
```

Comment:
Seems that when  proc_open is included in php_disabled_functions, the `wp db export` commands report this error.

    Error: Cannot do 'run_mysql_command': The PHP functions `proc_open()` and/or `proc_close()` are disabled. Please check your PHP ini directive `disable_functions` or suhosin settings.

Other wp-cli commands seem to work though.

Also this command works fine: `mysqldump -uroot chappress_dev > /tmp/wp-$(date +"%Y%m%d_%H%M%S").sql`

I can add some notes to troubleshooting section.